ui2/src/common/Button.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-import { Button } from 'antd';
+import { Button, Tooltip } from 'antd';
 import './Button.css';
 
-export const AppButton = ({ label, className, isLoading, ...props }) => (
-  <Button className={`${className} app-btn`} loading={isLoading} {...props}>
-    {label}
-  </Button>
+export const AppButton = ({ label, className, isLoading, tooltipText, ...props }) => (
+  <Tooltip title={tooltipText}>
+    <Button className={`${className} app-btn`} loading={isLoading} {...props}>
```

Comment:
if you will rename IsLoading to loading you could remove the loading={isLoading}

Reply:
done